homeassistant/components/panel_custom.py

```diff
@@ -41,7 +45,10 @@
         vol.Optional(CONF_URL_PATH): cv.string,
         vol.Optional(CONF_CONFIG): dict,
         vol.Optional(CONF_WEBCOMPONENT_PATH): cv.isfile,
```

Comment:
Webcomponent path should be part of the exclusive group. We should also add a small test to make sure this schema works.